packages/databyss-services/sources/actions.js

```diff
@@ -74,6 +87,22 @@ export function removeSourceFromCache(id) {
   }
 }
 
+export function fetchAllSources() {
```

Comment:
This is used in the `EditorPage.js` `useEffect` hook when looking up if a source exists. or if a new one has been added and must be cached. An alternative to this would be to load all the sources a page currently contains on the initial mount of that specific page. Either way, we must have all the sources cached if we want to compare our `editorState` to the `sourceProvider`.

Reply:
No we can remove this and use `fetchPageSources`, right?